University of Leicester/Leicester210.xml

```diff
@@ -0,0 +1,326 @@
+</desc>
+</item>
+<item xml:id="e7">
+<desc type="Correction">Scribal corrections written over erasures, e.g. <locus target="#7ra"></locus>, <locus target="25rb"></locus>, <locus target="#26rb"></locus>.</desc>
+</item>
+<item xml:id="e8">
+<desc>Writing exercises, minor notes crudely written by a recent hand (red and black ink)): e.g. <locus target="#1v"></locus>, <locus target="#88v"></locus>, <locus target="#90r"></locus>.</desc>
+</item>
+<item xml:id="e9">
+<desc>Recent drawings on <locus target="#1v"></locus> (black ink) and <locus target="#90v"></locus> (black and red/brown ink).</desc>
+</item>
+<item xml:id="e10">
+<desc type="findingAid">Scribal pen trial: fol. 90v.</desc>
+</item>
+<item xml:id="e11">
+<desc>Pencil doodles in the margin of some folios, e.g. <locus target="#14r"></locus>, <locus target="#16r"></locus>, and on the inner side of the wooden left board.</desc>
+</item>
+</list>
+</additions>
+ </physDesc>
+                  <history>
+                     <origin>
+                        <origPlace>
+                        <placeName ref="LOC3010Ethiop"/>
+                     </origPlace>
+<origDate notBefore="1350" notAfter="1450"  evidence="lettering">Mid-14th mid-15th century (?) (on the basis of the Palaeography).</origDate>
+                     </origin>
+                  </history>
+                  <additional>
+                     <adminInfo>
+                        <recordHist>
+                           <source>
+                             <note>Information from the Library on two write papers attached on the inner right board: the first one with the Coat-of-Arms of the University of Leicester and the text (half printed, half-hand written) ‘Library of the University College Leicester No. 091’, and in pencil above ‘MS 210’; the second one with the text (half printed, half-hand written) ‘This book was given by The Roljohns Bequest (?) 12394’.
+                               The manuscript is kept in a leather bookcase. On the front side of it, on a white field (paper?), it is written: (above) ‘A SPLENDID MANUSCRIPT upon VELLUM, written in the XIVth century,’; (below) ‘ARMENIAN MS., written on 89 leaves of stout vellum (6  by 5 in.), in double columns, one page containing curious drawings, original wooden boards half covered leather, preserved in a leather case of native make’.</note>
+                                <listBibl type="secondary">
+                                  <bibl>A very short and generical mention on the existing of this manuscript in the Library of the Leicester University College is in <ptr target="bm:Pearson1971Oriental"/>, <citedRange unit="page">117-118</citedRange>. Here we also read that the manuscript ‘has recently been examined and described by Prof. Ullendorff’ (<ptr target="bm:Pearson1971Oriental"/>, <citedRange unit="page">117</citedRange>).
```

Comment:
something is wrong here
the element `bibl `may only include `ptr `and `citedRange`, and also only one `ptr`, the way it is now is invalid

Try
```
<bibl><ptr target="bm:Pearson1971Oriental"/><citedRange unit="page">117-118</citedRange>. 
                                      <note>A very short and generical mention on the existing of this manuscript in the Library of the Leicester University College . Here we also read that the manuscript ‘has recently been examined and described by Prof. Ullendorff. </note>
                                  </bibl>
```